command-expansion-server/src/app.ts

```diff
@@ -282,10 +282,12 @@ app.post('/get-command-typescript', async (req, res, next) => {
 
     res.status(200).json({
       status: Status.SUCCESS,
-      typescriptFiles: [{
-        filePath: 'command-types.ts',
-        content: commandTypescript,
-      }],
+      typescriptFiles: [
```

Comment:
Is this just a formatting change?

Reply:
Yes it is, I was lazy and didn't want to split this change into a reformat commit. :P